src/components/storage/CategoriesTable.tsx

```diff
@@ -24,24 +25,26 @@ async function CategoriesTable({
     <NextIntlClientProvider
       messages={{ storage, ui } as Pick<Messages, 'storage' | 'ui'>}
     >
-      <Table>
-        <TableCaption>{t('categoryTableDescription')}</TableCaption>
-        <TableHeader>
-          <TableRow>
-            <TableHead>ID</TableHead>
-            <TableHead className='flex items-center justify-between'>
-              <span>{t('categoryName')}</span>
-              <span>{t('actions')}</span>
-            </TableHead>
-          </TableRow>
-        </TableHeader>
-        <TableBody>
-          {categories.map((category) => (
-            <CategoriesTableFormRow key={category.id} category={category} />
-          ))}
-          <CategoriesTableFormRow />
-        </TableBody>
-      </Table>
+      <ScrollArea className='w-full' orientation='horizontal'>
+        <Table>
+          <TableCaption>{t('categoryTableDescription')}</TableCaption>
+          <TableHeader>
+            <TableRow>
+              <TableHead>ID</TableHead>
+              <TableHead className='flex items-center justify-between'>
+                <span>{t('categoryName')}</span>
+                <span>{t('actions')}</span>
+              </TableHead>
+            </TableRow>
+          </TableHeader>
+          <TableBody>
+            {categories.map((category) => (
+              <CategoriesTableFormRow key={category.id} category={category} />
+            ))}
+            <CategoriesTableFormRow />
+          </TableBody>
+        </Table>
```

Comment:
```suggestion
          <TableCaption>{t('categoryTableDescription')}</TableCaption>
        </Table>
```
The caption is placed at the bottom anyway, so makes more sense to keep it there in the code too.